### ISSUE TITLE
Better landscape layout [with mockup]

### ISSUE THREAD

**Issue by BloodRaven0** [veloce/lichobile#1629]
Currently, the chessboard is really small in landscape mode, especially on devices with long screens. The culprit is the huge empty header that spans across the entire screen, when it doesn't need to. Here's the current UI:
![current](https://user-images.githubusercontent.com/42444745/112721173-33b76000-8f0b-11eb-88ed-231c550075f6.png)

And here's my proposed mockup for an improved UI:
![improved](https://user-images.githubusercontent.com/42444745/112721185-3c0f9b00-8f0b-11eb-9f1f-3a77648fa95b.png)

I realize that this might cause problems with more square-ish screens (on android TV maybe?), so it could either be adaptable, or have a toggle in the app settings.

**Comment by veloce** [comment]
Yes it's a good idea but needs to be tested on various screen size. No toggle wanted here, but we could detect screen ratio.
